mirdata/beatles.py

```diff
@@ -115,6 +191,12 @@ def load(data_home=None):
 
 
 def _load_beats(beats_path):
+    """Private function to load beat data for a track
+
+    Args:
+        beats_path (str):
+
```

Comment:
Add Returns:

Reply:
same comment for private loaders below

Reply:
Let me work on the private in the following PR.